packages/components/src/components/typography/Paragraph/index.tsx

```diff
@@ -47,7 +46,7 @@ const Paragraph = styled.div<Props>`
         `}
 `;
 
-interface Props extends React.HTMLAttributes<HTMLDivElement> {
+interface PProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
nit: ParagraphProps sounds better to me

Reply:
Even though the component is called `P`? I can see how `PProps` looks weird, but it would be a deviation from the pattern.

Reply:
I personally would have renamed  P component to Paragraph, but both are fine, let's leave it at that 👌